src/components/Template/Template.stories.tsx

```diff
@@ -174,251 +172,171 @@ const columnStylesComplex = [
+              />
+            </FormField>
+            <FormRow>
               <FormField>
                 <TextInput
-                  helperText="Make sure to complete this field."
-                  id="username"
-                  labelText="Username"
-                  isRequired
+                  id="phone-field"
+                  labelText="Phone Field"
+                  type="tel"
                 />
               </FormField>
               <FormField>
-                <TextInput
-                  helperText="Make sure to complete this field."
-                  id="password"
-                  labelText="Password"
-                  isRequired
-                />
+                <TextInput id="url-field" labelText="URL Field" type="url" />
               </FormField>
-              <FormRow>
-                <FormField>
-                  <TextInput
-                    id="phone-field"
-                    labelText="Phone Field"
-                    type="tel"
-                  />
-                </FormField>
-                <FormField>
-                  <TextInput id="url-field" labelText="URL Field" type="url" />
-                </FormField>
-                <FormField>
-                  <TextInput id="age-field" labelText="Age" type="number" />
-                </FormField>
-              </FormRow>
               <FormField>
-                <Button id="submit">Submit</Button>
+                <TextInput id="age-field" labelText="Age" type="number" />
               </FormField>
-            </Form>
-            <FeaturedContent
-              imageProps={{
-                alt: "",
-                src: getPlaceholderImage("smaller"),
-                position: "end",
-                width: "",
-              }}
-              isFullWidth
-              my="l"
-              textContent={
-                <>
-                  <Heading size="heading5">Sit Dapibus Elit</Heading>
-                  Donec id elit non mi porta gravida at eget metus. Nulla vitae
-                  elit libero, a pharetra augue. Cum sociis natoque penatibus et
-                  magnis dis parturient montes, nascetur ridiculus mus. Cras
-                  mattis consectetur purus sit amet fermentum.
-                </>
-              }
-            />
-            <Table
-              columnHeaders={columnHeadersAlt}
-              columnStyles={columnStylesComplex}
-              id="table-horizontal-scrolling-wo-row-headers"
-              isScrollable
-              showRowDividers
-              showTitleText={false}
-              tableData={tableDataAlt}
-              titleText="Table with horizontal scrolling and no row headers"
-              useRowHeaders
-            />
-          </TemplateContentPrimary>
-          <TemplateContentSidebar>
+            </FormRow>
+            <FormField>
+              <Button id="submit">Submit</Button>
+            </FormField>
+          </Form>
+          <Table
```

Comment:
Unrelated issue: this table does not scroll properly at 600px screen width

Reply:
Hmm, for me, the table scrolls at 600px, but you're right that it doesn't look great. 

@bigfishdesign13 & @EdwinGuzman, will this be resolved with the new md breakpoint? I believe it's 768px, meaning this would be one column up until then.

Reply:
Container queries would resolve this issue. The `Table` will not use a fixed first column for mobile viewports, but what you are seeing can happen when it is dropped in a narrow space. The new mobile breakpoint of 768px will help, but it will still not be ideal.

Another way to improve the UX here would be to turn off the row headers on the `Table` when the viewport width drops below `960px`.

Reply:
Yes, good points. Won't do anything about it here though.